src/richie/apps/core/templates/menu/footer_menu.html

```diff
@@ -8,12 +8,26 @@
             {% if child.sibling %} body-footer__menu__list__item--sibling{% endif %}
             {% if child.descendant %} body-footer__menu__list__item--descendant{% endif %}">
             <a class="body-footer__menu__list__item__link" href="{{ child.attr.redirect_url|default:child.get_absolute_url }}">{{ child.get_menu_title }}</a>
```

Comment:
can you wrap this in a block so it can be removed by overriding the template and not rewriting the whole thing?

Reply:
What line(s) are we talking about to move to a block ?

Also this for a usecase where someone would create a "myproject/menu/footer_menu.html" which inherits from richie footer_menu and then change base.html to use this path on templatetag "show_menu_belowid" in footer ?

Reply:
Finally we let down this since it is simpler to override the whole template since it can be overriden directly from "base.html".